SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -280,6 +281,69 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	}
 }
 
+void USpatialNetDriver::CreateServerWorkerEntity(int AttemptCounter)
```

Comment:
Hmm, this being in the NetDriver is iffy. I think this potentially should sit elsewhere.

Reply:
I think it should at least sit in the Sender since it's sending the entity. The call to this function can sit in the NetDriver.